apps/frontend/components/LabelSwitch/LabelSwitch.tsx

```diff
@@ -1,24 +1,108 @@
-import { Grid, Switch } from '@mui/material';
-import { ChangeEvent, ReactNode } from 'react';
+import {
+  Stack,
+  Switch,
+  SwitchProps,
+  Typography,
+  useTheme,
+} from '@mui/material';
+import { styled } from '@mui/material/styles';
+import { ChangeEvent } from 'react';
+import { FormattedMessage } from 'react-intl';
+
+import { colors } from 'theme/muiTheme';
+
+const IOSSwitch = styled((props: SwitchProps) => (
```

Comment:
Are you sure we needed to change the component completely here?

Reply:
This is basically the `Switch` component from `mui`. It just has styles applied to match the designs